src/angular/9-form-value-changes/form-value-changes.md

```diff
@@ -72,7 +72,7 @@ The ReactiveForms API makes it easy for us to change our FormControls as needed.
 ## P1: Technical Requirements
 
 1. Subscribe to the `state` and `city` formControl value changes and log the resulting value to the console.
-2. Unsubscribe from Subscription created in step 1 in the `ngOnDestroy` function
+2. Unsubscribe from Subscriptions at `restaurant.component` in the `ngOnDestroy` function
```

Comment:
Maybe say "Subscriptions within `restaurant.component`" instead?

Reply:
absolutely if it sounds better to you. I'll commit this change